tests/cuda/CMakeLists.txt

```diff
@@ -15,5 +15,6 @@ vecmem_add_test( cuda_allocators test_cuda_allocators.cpp
    LINK_LIBRARIES vecmem::core vecmem::cuda )
 
 # Tests for the core library's device vector types, using the CUDA backend.
-vecmem_add_test( cuda_containers test_cuda_containers.cu
+vecmem_add_test( cuda_containers test_cuda_containers.cpp
+   test_cuda_containers_kernels.cu
```

Comment:
Just one comment on this one. :stuck_out_tongue: Please add `test_cuda_Containers_kernels.cuh` to the list here. That will allow some IDEs (like Xcode and VisualStudio) to list the files belonging to the executable correctly.

I'll still need to add some CMake code to make those IDEs like our setup, but at least I won't have to do this later on then. :smile: